src/viking.js

```diff
@@ -1,14 +1,79 @@
+
+
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor (health, strength){
+    this.strength = strength;
+    this.health = health;
+  }
+  attack(){
+    return this.strength
+  }
+  receiveDamage(damage){
+    this.health -= damage
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength){
+    super (name, health, strength)
+    this.name = name;
+   }
+   receiveDamage(damage){
+    this.health -= damage;
+    if(this.health === 0 ){
+       return `${this.name} has died in act of combat`
+    }else if(this.health > 0){
+      return `${this.name} has received ${damage} points of damage`
+    }
+  }
+  battleCry(){
+    return `Odin Owns You All!`
+  }
+
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+  
+  receiveDamage(damage){
+    this.health = this.health - damage
+    if(this.health == 0 ){
+      return `A Saxon has died in combat`
+   }else if(this.health > 0){
+     return `A Saxon has received ${damage} points of damage`
+   }
+  }
 
+   
+}
+   
 // War
-class War {}
+class War {  
+  constructor (vikingArmyPorperty,saxonArmyProperty){
+       this.vikingArmyPorperty = vikingArmyPorperty;
+       this.saxonArmyProperty = saxonArmyProperty;
```

Comment:
En esta parte te recomiendo que sean arreglos que contengan a cada objeto de los enemigos, y después con push agregas elementos:
`this.vikingArmy = []
    this.saxonArmy = []`